bin/near-cli.js

```diff
@@ -180,6 +180,11 @@ yargs // eslint-disable-line
         desc: 'Expected top-level account for a network',
         type: 'string'
     })
+    .option('verbose', {
+        desc: 'Prints out verbose output',
+        type: 'boolean',
+        default: true
```

Comment:
I'm strongly in favor of making this `default: false`. That's much more standard in CLIs – by default, you don't get inundated with output. You have to add `--verbose` or `-v` for that.

Right now, this will be backwards – inundation by default, and you have to type what? `--verbose=false`? in order to turn it off? I don't like it.

Let's also add the `-v` alias, please!

Reply:
While I agree that generally the verbosity is usually opt-in, I feel that Corwin and Ash's direction (and the ticket that I believe Jane will work on after this) is explicitly to be _more loud_ and not less loud.
https://github.com/near/devx/issues/212

Reply:
Are we fine with the options disappearing from everywhere though? It's a change from existing behaviour, and some folks don't like change

Reply:
done